doc/api/resources/orders.rst

```diff
@@ -609,13 +609,17 @@ Fetching individual orders
 Order ticket download
 ---------------------
 
+.. versionchanged:: 4.10
+
+   The API now supports ticket downloads for pending orders if allowed by the event settings.
+
 .. http:get:: /api/v1/organizers/(organizer)/events/(event)/orders/(code)/download/(output)/
 
    Download tickets for an order, identified by its order code. Depending on the chosen output, the response might
    be a ZIP file, PDF file or something else. The order details response contains a list of output options for this
    particular order.
 
-   Tickets can be only downloaded if the order is paid and if ticket downloads are active. Note that in some cases the
+   Tickets can be only downloaded if the order is paid (depending on event settings) and if ticket downloads are active. Note that in some cases the
```

Comment:
I think if you do not know that tickets for pending orders are possible, the part „depending on event settings“ is a bit confusing as there is in „what is the alternative to paid orders?“. I am not 100% sure though as this setting does not seem to be available through the GUI – then I guess swapping around „can be only downloaded“ to „can only be downloaded“ is enough.

```suggestion
   Tickets can only be downloaded if ticket downloads are active and – depending on event settings – the order is either paid or pending. Note that in some cases the
```